src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/models/probationoffendersearch/Offender.kt

```diff
@@ -33,7 +33,7 @@ data class Offender(
           deliusCrn = otherIds.crn,
         ),
       pncId = otherIds.pncNumber,
-      hmppsId = otherIds.crn,
+      hmppsId = if (otherIds.crn?.isNotEmpty() == true) otherIds.crn else otherIds.nomsNumber,
```

Comment:
Would we want to return nonsNumber as hmppsId?  Also would probation search return person without CRN? Perharps you are looking to update POSPrisoner(person search from Prisoner serach) to return hmppsId?